addons/mail/static/src/components/chat_window_header/chat_window_header.scss

```diff
@@ -13,48 +11,13 @@
 }
 
 .o_ChatWindowHeader_command {
-    padding: map-get($spacers, 0) map-get($spacers, 3);
-    display: flex;
-    height: map-get($sizes, 100);
-    align-items: center;
-
     &:hover {
         background-color: rgba($black, 0.1);
     }
 }
 
-.o_ChatWindowHeader_commandBack {
-    margin-right: map-get($spacers, 2);
-}
-
-.o_ChatWindowHeader_item {
-    margin: map-get($spacers, 0) map-get($spacers, 1);
-
-    &.o_ChatWindowHeader_rightArea {
-        margin-right: map-get($spacers, 0);
-    }
-
-    &:first-child {
-        margin-left: map-get($spacers, 3);
-
-        &.o_ChatWindowHeader_command {
-            margin-left: map-get($spacers, 0); // no margin for commands
-        }
-    }
-
-    &.o_ChatWindowHeader_rightArea:last-child .o_ChatWindowHeader_command {
-        margin-right: map-get($spacers, 0); // no margin for commands
-    }
-}
-
-.o_ChatWindowHeader_name {
-    max-height: map-get($sizes, 100);
-}
-
-.o_ChatWindowHeader_rightArea {
-    display: flex;
-    height: map-get($sizes, 100);
-    align-items: center;
+.o_ChatWindowHeader_item:first-child.o_ChatWindowHeader_command {
+    margin-left: map-get($spacers, 0); // no margin for commands
```

Comment:
I haven't tested yet in practice, but this looks dead code: no `item` that is also a `command` is ever a `first-child` with current xml.

Maybe the `item` classname is mistakenly missing on `commandBack`?

Reply:
You're right, this has no effect on any of the `commands`, they all have a `m-0` class anyway (except commandBack which I needs its right margin)